<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="userFrame">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="header_user_frame">
        <property name="styleSheet">
         <string notr="true">background-color:transparent;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item alignment="Qt::AlignHCenter">
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item alignment="Qt::AlignVCenter">
             <widget class="QLineEdit" name="search_input">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>360</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>360</width>
                <height>36</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>15</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	color:rgb(121, 121, 121);
background-color:white;
border: 1px solid rgb(199, 199, 199);
border-radius: 10px;
border-bottom: 3px solid rgb(91, 91, 91)
}

QLineEdit:hover{background-color:rgb(238, 234, 233);color:rgb(181, 178, 177)}
</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="placeholderText">
               <string>Pesquisa</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QFrame" name="search_frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="btnSearch">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>45</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>45</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">

QPushButton { color: rgb(243, 239, 238);
	border-radius:15px;		
	border: 4px solid #00BF63;
	background-color:#00BF63;
 }

QPushButton:hover{background-color:rgb(255, 240, 238);color:#00BF63}

QPushButton:pressed{
 padding-left: 5px;
 padding-top:5px;
	background-color: rgb(227, 227, 227);
 
}</string>
              </property>
              <property name="text">
               <string>Buscar</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/search.svg</normaloff>:/icons/search.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>27</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="infoFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="btnDetails">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>45</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>45</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">

QPushButton { color: rgb(243, 239, 238);
	border-radius:15px;		
	border: 4px solid #00BF63;
	background-color:#00BF63;
 }

QPushButton:hover{background-color:rgb(255, 240, 238);color:#00BF63}

QPushButton:pressed{
 padding-left: 5px;
 padding-top:5px;
	background-color: rgb(227, 227, 227);
 
}</string>
              </property>
              <property name="text">
               <string>Detalhes</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/info.svg</normaloff>:/icons/info.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>27</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="cadFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="btnCad">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>45</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>45</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">

QPushButton { color: rgb(243, 239, 238);
	border-radius:15px;		
	border: 4px solid #00BF63;
	background-color:#00BF63;
 }

QPushButton:hover{background-color:rgb(255, 240, 238);color:#00BF63}

QPushButton:pressed{
 padding-left: 5px;
 padding-top:5px;
	background-color: rgb(227, 227, 227);
 
}</string>
              </property>
              <property name="text">
               <string>Cadastrar</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/user-plus.svg</normaloff>:/icons/user-plus.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>27</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="text_Frames">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #057A3A;
    border-top-right-radius: 15px;  
    border-bottom-right-radius: 15px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255)</string>
           </property>
           <property name="text">
            <string>Usuários cadastrados</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255)</string>
           </property>
           <property name="text">
            <string>Últimos registros</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="info_user">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:9px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTableView" name="usersList">
           <property name="styleSheet">
            <string notr="true">background-color:white;	
border:3px solid #057A3A;
border-radius:10px

</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="logsReg">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white;	
border:3px solid #057A3A;
border-radius:10px

</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="createFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../feather/resource01.qrc"/>
 </resources>
 <connections/>
</ui>
